tools/cli/main.js

```diff
@@ -616,7 +616,7 @@ Fiber(function () {
   // tight timetable for 1.0 and there is no advantage to doing it now
   // rather than later. #ImprovingCrossVersionOptionParsing
 
-  var isBoolean = { "--help": true, "--unsafe-perm": true };
```

Comment:
I agree that it's cleaner; I'm not a fan of environment variables in general, but I guess I don't care either way.  However, this will break anyone (again) who has made the change today to use `--unsafe-perm`.